core/vm/memory.go

```diff
@@ -55,10 +57,9 @@ func (m *Memory) Set32(offset uint64, val *uint256.Int) {
 	if offset+32 > uint64(len(m.store)) {
 		panic("invalid memory: store empty")
 	}
-	// Zero the memory area
-	copy(m.store[offset:offset+32], []byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0})
-	// Fill in relevant bits
-	val.WriteToSlice(m.store[offset:])
+	var buf [32]byte
```

Comment:
FYI: we have `zeroes` global variable in this file

Reply:
Good point. I changed to use that instead.